### DISCUSSION TITLE
Are the 4.0 code signing changes valid and safe?

### DISCUSSION THREAD

**Discussion by sporkman** [SelfControlApp/selfcontrol#725] [Q&A] [answered]
Hi,

Could anyone from the project confirm this change in the code signer's sig is correct?

If so, could a change like that be noted in future release notes?

Thanks,

Charles
<img width="766" alt="Screen Shot 2021-10-05 at 4 06 22 PM" src="https://user-images.githubusercontent.com/510750/136095423-37d4111b-0d14-4497-b0b2-696cd61b96a8.png">

**Comment by cstigler** [comment, marked as answer]
Huh, was not expecting this change to trigger any warnings, but you are the second person to notice (see #721) so apparently I was wrong. Yep, this is a valid and expected change, no cause for concern.

My old developer account got nerfed by a bureaucratic error at Apple, hence the change. It's a big hassle so no plans to switch ever again :)

**Reply by sporkman** [reply]
Yeah, I run Little Snitch (from obdev.at) and while it's basically a little per-app firewall, it does record code signatures for every app for which it allows outbound (and I guess inbound) access. If the sig changes, it wants a confirmation.
